README.md

```diff
@@ -85,6 +94,8 @@ To run `lint` and `prettier` in both the frontend and backend and unit tests in
 ```bash
 npm run check:all
 ```
+> Note: If you recieve a workspace error, run `npx next telemetry disable` 
```

Comment:
Maybe note that this would be a warning that does not prevent the service from running but this command can be run to remove that warning